.github/actions/Tests/Unattended/test_unattended.py

```diff
@@ -0,0 +1,149 @@
+                        get_kibana_password()), 
+                        verify=False)
+    return (resp.status_code)
+
+def get_wazuh_api_status():
+
+    protocol = 'https'
+    host = 'localhost'
+    port = 55000
+    user = 'wazuh'
+    password = 'wazuh'
+    login_endpoint = 'security/user/authenticate'
+
+    login_url = f"{protocol}://{host}:{port}/{login_endpoint}"
+    basic_auth = f"{user}:{password}".encode()
+    login_headers = {'Content-Type': 'application/json',
+                    'Authorization': f'Basic {b64encode(basic_auth).decode()}'}
+    response = requests.get(login_url, headers=login_headers, verify=False)
+    token = json.loads(response.content.decode())['data']['token']
+    requests_headers = {'Content-Type': 'application/json',
+                        'Authorization': f'Bearer {token}'}
+    response = requests.get(f"{protocol}://{host}:{port}/?pretty=true", headers=requests_headers, verify=False)
+
+    return response.json()['data']['title'] 
+
+# ----------------------------- Tests ----------------------------- 
+
+
+
+def test_check_wazuh_manager_authd():
+    assert check_call("ps -xa | grep wazuh-authd | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_db():
+    assert check_call("ps -xa | grep wazuh-db | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_execd():
+    assert check_call("ps -xa | grep wazuh-execd | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_analysisd():
+    assert check_call("ps -xa | grep wazuh-analysisd | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_syscheckd():
+    assert check_call("ps -xa | grep wazuh-syscheckd | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_remoted():
+    assert check_call("ps -xa | grep wazuh-remoted | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_logcollec():
+    assert check_call("ps -xa | grep wazuh-logcollec | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_monitord():
+    assert check_call("ps -xa | grep wazuh-monitord | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_modulesd():
+    assert check_call("ps -xa | grep wazuh-modulesd | grep -v grep", shell=True) != ""
+
+def test_check_wazuh_manager_apid():
+    assert check_call("ps -xa | grep wazuh-apid | grep -v grep", shell=True) != ""
+
+def test_check_filebeat_process():
+    assert check_call("ps -xa | grep \"/usr/share/filebeat/bin/filebeat\" | grep -v grep", shell=True) != ""
+
+def test_check_elasticsearch_process():
+    assert check_call("ps -xa | grep \"/usr/share/elasticsearch/jdk/bin/java\" | grep -v grep | cut -d \" \" -f15", shell=True) != ""
+
+def test_check_kibana_process():
+    assert check_call("ps -xa | grep \"/usr/share/kibana/bin/../node/bin/node\" | grep -v grep", shell=True) != ""
+
+def test_check_elasticsearch_cluster_status():
+    assert get_elasticsearch_cluster_status() != "red"
```

Comment:
This will pass a cluster with yellow state, i think we must be more strict with this state